tests/test_other.py

```diff
@@ -450,15 +451,15 @@ def test_combining_object_files_from_archive(self):
     self.assertExists('combined.a')
 
     # Combining library archive into an object should yield a valid object, using the `-r` flag
-    self.run_process([EMCC, '-r', '-o', 'combined.o', '-Wl,--whole-archive', 'combined.a'])
+    self.run_process([EMXX, '-r', '-o', 'combined.o', '-Wl,--whole-archive', 'combined.a'])
```

Comment:
why change it here?

Reply:
You mean because we are using `-r` here?  I guess technically we might not need it, but I still think its more correct because `combined.a` contains objects at are C++ objects, not C objects.